plugins/modules/vmware_dvs_portgroup.py

```diff
@@ -624,20 +1349,37 @@ def main():
                     uplink_teaming_override=False,
                     vendor_config_override=False,
                     vlan_override=False
-                )
-            )
+                ),
+                aliases=['port_policy']
+            ),
+            netflow_enabled=dict(type='bool', default=False),
+            block_all_ports=dict(type='bool', default=False),
+            # TODO: traffic shaping
+            # TODO: traffic filtering and marking
+            # NOTE: The below parameters are deprecated starting from Ansible collection 2.0.0
+            vlan_trunk=dict(
+                type='bool',
+                default=False,
+                removed_in_version='2.0.0',
+                removed_from_collection='community.vmware',
+                ),
+            portgroup_type=dict(
+                type='str', choices=['earlyBinding', 'lateBinding', 'ephemeral'], removed_in_version='2.0.0',
+                removed_from_collection='community.vmware',
+            ),
         )
     )
 
-    module = AnsibleModule(argument_spec=argument_spec,
-                           mutually_exclusive=[
-                               ['vlan_trunk', 'vlan_private']
-                           ],
-                           supports_check_mode=True)
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        mutually_exclusive=[
+            ['vlan_id', 'vlan_trunk_range', 'private_vlan_id']
+        ],
+        supports_check_mode=True
+    )
 
     vmware_dvs_portgroup = VMwareDvsPortgroup(module)
     vmware_dvs_portgroup.process_state()
 
-
 if __name__ == '__main__':
```

Comment:
```suggestion

if __name__ == '__main__':
```
CI breaks because of a pep8 issue:

E305: expected 2 blank lines after class or function definition, found 1